sqlbp/client.go

```diff
@@ -0,0 +1,190 @@
+package sqlbp
+
+import (
+	"context"
+	"database/sql"
+	"database/sql/driver"
+	"time"
+
+	"github.com/go-kit/kit/metrics"
+	"github.com/jmoiron/sqlx"
+	"github.com/luna-duclos/instrumentedsql"
+	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/reddit/baseplate.go/metricsbp"
```

Comment:
style nit: add a blank line before this line. see: https://github.com/reddit/baseplate.go/blob/master/Style.md#import-groups